index.html

```diff
@@ -185,6 +186,42 @@ <h1 class="multi-post-header">Multi-Post Stories</h1>
                 </div>
             </section>
         </section>
+        <form action="https://formspree.io/f/xjvlzgoa" method="post">
+            <h2 class="form-header">CONTACT US</h2>
+            <P class="form-paragraph">If you have an application you are interested
+                in developing, a feature that you need built or
+                a project that needs coding. I&#x27;d love to help with it.
+            </P>
+            <ul>
+                <li>
+                    <label for="name"></label>
+                    <input type="text" name="user_name" id="name" minlength="2" maxlength="30" required placeholder="Text Input">
+                </li>
+                <li>
+                    <label for="email"></label>
+                    <input type="email" name="user_email" id="email" placeholder="jeremy@gmail.com" required>
+                </li>
+                <li>
+                    <label for="msg"></label>
+                    <textarea name="user_message" id="msg" placeholder="Enter text here" required maxlength="500"></textarea>
+                </li>
+            </ul> 
```

Comment:
- [ ] Kindly make sure you used the correct `text-color` on inputs like on the Figma template as some of them have the same **color**  for `text` and `background` and when typing any message, it cannot be visible.
- [ ] Make sure you also used the correct `width` on all form's elements and `margin)` between form's elements and `padding` around text in inputs fields to make your design stick to the Figma template.